src/Storage/AbstractSessionArrayStorage.php

```diff
@@ -67,44 +67,39 @@ public function init($input = null)
     /**
      * Get Offset
      *
-     * @param  mixed $key
      * @return mixed
      */
-    public function __get($key)
+    public function __get(mixed $key)
```

Comment:
I don't think this is really valid :thinking: 

I know this was done in an automated way, but the parent interface is much stricter.

Anyway, OK as-is to prevent regressions in downstream implementors.